lib/events/codes.go

```diff
@@ -762,6 +762,25 @@ const (
 	// join_token.bound_keypair.join_state_verification_failed.
 	BoundKeypairJoinStateVerificationFailedCode = "TBK003W"
 
+	// SCIMResourceProvisionSuccessCode indicates a resource was successfully
+	// provisioned via SCIM
+	SCIMResourceProvisionSuccessCode = "TSCIM001I"
```

Comment:
The SCIM don't have provision  API can we rename it to Create to be consistent with `SCIMResourceDelete` ?